sklearn/metrics/cluster/unsupervised.py

```diff
@@ -47,13 +52,28 @@ def silhouette_score(X, labels, metric='euclidean', sample_size=None,
         <sklearn.metrics.pairwise.pairwise_distances>`. If X is the distance
         array itself, use ``metric="precomputed"``.
 
+    blockwise: {'auto', True, False}
+        Enables blockwise computation of the distance matrix.
+        If false, the full distance matrix is computed yielding in fast
+        computation but high memory consumption. If True, it computes
+        clusterwise distance matrices, dividing memory consumption by
+        approximately the squared number of clusters. The latter allows
+        parallelization through ``n_jobs`` parameter.
+        Default is 'auto' that choses the method to use depending on some
+        heuristics.
+
     sample_size : int or None
-        The size of the sample to use when computing the Silhouette Coefficient 
-        on a random subset of the data. 
+        The size of the sample to use when computing the Silhouette Coefficient
+        on a random subset of the data.
         If ``sample_size is None``, no sampling is used.
 
+    n_jobs : integer, optional
+        The number of CPUs to use to do the computation. -1 means
+        'all CPUs'. This option can only be used with ``method="blockwise"``.
```

Comment:
This docstring hasn't been updated to reflect the change of naming in the arguments.